sdk/src/secp256k1_instruction.rs

```diff
@@ -911,7 +911,7 @@ pub fn construct_eth_pubkey(
 
 /// Verifies the signatures specified in the secp256k1 instruction data.
 ///
-/// This is same the verification routine executed by the runtime's secp256k1 native program,
+/// This is same as the verification routine executed by the runtime's secp256k1 native program,
```

Comment:
nit: Please update `This is same as the ...` to `This is the same as the ...`

Reply:
@steviez sure, just updated!